app/jobs/notifications/send_notification_job.rb

```diff
@@ -9,7 +9,8 @@ def perform notification, owner, recipient, content, invoice, click_action
     end
     text = I18n.t("notifications.#{content}", user_name: owner.name, invoice_name: invoice.name)
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/LineLength
**Severity**: INFO
**File**: app/jobs/notifications/send_notification_job.rb L10

Line is too long. [96/80]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*